apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/ProportionalRandomJitterTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright 2018 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Linq;
+using Xunit;
+
+namespace Google.Cloud.Spanner.V1.Tests
+{
+    public class ProportionalRandomJitterTests
+    {
+        [Fact]
+        public void ValuesWithinRange()
+        {
+            // We might as well use the example from the docs...
+            double randomProportion = 0.1;
+            TimeSpan delay = TimeSpan.FromMinutes(60);
+
+            var jitter = new ProportionalRandomJitter(randomProportion);
+
+            var delays = Enumerable.Range(0, 10000)
+                .Select(_ => jitter.GetDelay(delay))
+                .ToList();
+            var min = delays.Min();
+            var max = delays.Max();
+
+            // We don't mind if rounding reduces the lowest value to just under 54 minutes
```

Comment:
Just curious: in this line rounding refers to the rounding of ticks to minutes? I'm just not sure how this could happen, I mean falling under 54 (I know it's not statistically relevant for our use case)